resources/authgear/templates/en/web/authflowv2/setup_totp.html

```diff
@@ -31,7 +31,7 @@ <h2 class="screen-description whitespace-pre-wrap">{{ template "v2-setup-totp-de
 
     <a
       class="primary-btn w-full mt-10"
-      href="{{ call $.MakeURL "setup_totp_verify" }}"
+      href="{{ call $.MakeURL "setup_totp" "x_action" "submit"}}"
```

Comment:
Instead of using `x_action` (which is already a name for submit button, let's use `q_setup_totp_step=verify`. Since it starts with `q_`, it is eliminated automatically.

If `q_setup_totp_step=verify`, you render the verify screen. Otherwise you render the qr code screen.

Reply:
Fixed in ad6c8d2